launchable/utils/session.py

```diff
@@ -0,0 +1,70 @@
+import click
+import os
+import json
+from pathlib import Path
+from typing import Optional
+
+session_file_path = Path('~/.config/launchable/session.json').expanduser()
+
+
+class SessionError(Exception):
+    pass
+
+
+def read_session(build_name: str) -> Optional[str]:
+    if not session_file_path.exists():
+        raise SessionError(
+            "No session file. Please write session {}".format(session_file_path))
+
+    with session_file_path.open('r') as json_file:
+        try:
+            data = json.load(json_file)
+            return data.get("{}:{}".format(build_name, os.getsid(os.getpid())))
+        except json.JSONDecodeError as e:
+            raise SessionError(
+                "Invalid session file. Please remove {}".format(session_file_path))
```

Comment:
So again I'm thinking what situation would land here. Probably write has failed because of out of disk space? Any other situations? Maybe I'm missing that scenario, but I can't think of a situation where instructing users to remove this file is the right diagnosis. Especially given that this tends to happen on a CI system where such access is difficult.

Also, I suggest we chain the nested exception `e` so that it gets displayed. Imagine a random user dropping by our community and ask you he saw this error message and how he can solve it? Especially in case of an error, we need every bit of information to troubleshoot.